backend/benefit/applications/tests/test_alteration_api.py

```diff
@@ -0,0 +1,726 @@
+import faker
```

Comment:
Whoah a looot of test 👍

Reply:
To be fair, most of these aren't new to this PR, just moved out of `test_application_api.py` because it seemed prudent to collect them into their own file 😅